compiler/rustc_span/src/symbol.rs

```diff
@@ -2325,13 +2325,15 @@ pub mod sym {
     ///
     /// The first few non-negative integers each have a static symbol and therefore
     /// are fast.
-    pub fn integer<N: TryInto<usize> + Copy + ToString>(n: N) -> Symbol {
+    pub fn integer<N: TryInto<usize> + Copy + itoa::Integer>(n: N) -> Symbol {
         if let Result::Ok(idx) = n.try_into() {
```

Comment:
Unrelated to the PR, but why does this use `Result::Ok` instead of just `Ok`? D:

Reply:
No idea. The [original commit](https://github.com/rust-lang/rust/commit/9c7d28d4fdd95bcd6062fb82a2dd2f280bda3e72#diff-9589e2d82ff438a80167897af5df86abc549367645c129d3447907c9a83c9480R979-R986) even had `Option::Some` :sweat_smile:.